tests/unit_tests/docmaps/v2/codecs/preprint_test.py

```diff
@@ -2,56 +2,75 @@
 from data_hub_api.docmaps.v2.codecs.preprint import (
     get_docmap_preprint_assertion_item,
     get_docmap_preprint_input,
-    get_docmap_preprint_input_with_published_and_tdmpath
+    get_docmap_preprint_input_with_published_and_meca_path,
+    get_meca_path_content
 )
 from tests.unit_tests.docmaps.v2.test_data import (
     DOI_1,
     MANUSCRIPT_VERSION_1,
     PREPRINT_LINK_1,
     PREPRINT_VERSION_1,
-    TDM_PATH_1
+    MECA_PATH_1
 )
 
 
 class TestGetDocmapPreprintInput:
-    def test_should_populate_published_and_tdm_path_if_detailed_param_true(self):
-        result = get_docmap_preprint_input_with_published_and_tdmpath(
+    def test_should_populate_docmap_preprint_input(self):
+        result = get_docmap_preprint_input(
             manuscript_version=MANUSCRIPT_VERSION_1
         )
         assert result == {
             'type': 'preprint',
             'doi': DOI_1,
             'url': PREPRINT_LINK_1,
             'versionIdentifier': PREPRINT_VERSION_1,
-            'published': date.fromisoformat('2021-01-01').isoformat(),
```

Comment:
We removed the `published` date as an input as well?

Reply:
Yes we are not including 'published' in the input according to accepted sample and current the DocMaps.